ci/fih-tests_run.sh

```diff
@@ -19,7 +19,7 @@ set -e
 pushd .. &&\
    git clone https://git.trustedfirmware.org/TF-M/trusted-firmware-m.git &&\
    pushd trusted-firmware-m &&\
-   git checkout TF-Mv1.7.0 &&\
+   git fetch https://review.trustedfirmware.org/TF-M/trusted-firmware-m refs/changes/09/18309/1 && git checkout FETCH_HEAD &&\
```

Comment:
I have updated the patch in TF-M. You can check out the latest version with this command:
`git fetch https://review.trustedfirmware.org/TF-M/trusted-firmware-m refs/changes/09/18309/2 && git checkout FETCH_HEAD`